docs/guide/mqtt.md

```diff
@@ -74,6 +74,7 @@ There are some custom apis that can be called that are not part of Zwave Client:
   - `lastUpdate`: Timestamp of latest event received from OZW
   - `status`: Client status. Could be: 'driverReady', 'connected', 'scanDone', 'driverFailed', 'closed'
   - `cntStatus`: Controller status received from ozw notifications controller command. If inclusion/exclusion is running it would be `Waiting`
+- `refreshValues`: Queries all actuator and sensor values of all node endpoints
```

Comment:
This is not needed as it is a zwave api

Reply:
How do people know it exists, then? Someone on slack yesterday asked if it existed as they'd been looking through the docs.

Reply:
because that api supports all zwave apis by default, in the list I only specify the ones that are not from zwave

Reply:
We make a list of the supported ones too, If you want to

Reply:
Might be a good idea. I think the mqtt documentation in general is a little hard to understand. May be worth having a second look at the whole thing with an eye towards an average user. I removed it from the PR for now.

Reply:
@blhoward2 I'm open to suggestion/help about how to improve it :)

Reply:
mqtt is an odd beast. It is fairly advanced but every now and then a normal user needs to publish a message (and HA lets you do so within an automation as an action)